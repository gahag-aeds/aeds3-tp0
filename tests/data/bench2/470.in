470 235
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 73 99 18 9 0 49 48 41 23 48 41 88 49 6 80 75 36 6 17 68 64 43 50 0 91 35 77 58 6 79 22 31 78 40 92 31 89 92 72 12 41 13 52 90 71 32 17 59 90 34 28 7 30 30 7 21 65 84 31 72 64 53 55 94 45 48 77 34 40 49 98 33 15 50 76 38 34 93 98 76 80 78 83 62 8 42 83 25 79 66 49 95 71 5 41 16 5 19 2 97 68 0 83 35 2 59 74 88 4 24 16 36 2 52 98 62 94 33 39 25 51 89 20 22 46 62 90 51 33 92 0 1 44 83 89 98 42 63 38 99 39 55 35 93 7 86 55 53 19 94 
Sub 284 439
Sum 2 157
Sub 234 389
Max 278 433
Sub 198 353
Sum 176 331
Sub 106 261
Sum 155 310
Add 161 316
Sum 161 316
Add 135 290
Min 260 415
Add 295 450
Max 286 441
Add 69 224
Sum 272 427
Sub 254 409
Max 178 333
Add 257 412
Max 55 210
Add 163 318
Min 104 259
Sub 266 421
Sum 166 321
Add 125 280
Min 219 374
Sub 220 375
Min 16 171
Sub 208 363
Max 34 189
Sub 288 443
Max 177 332
Add 67 222
Max 208 363
Add 141 296
Min 264 419
Add 69 224
Max 278 433
Sub 136 291
Max 262 417
Add 157 312
Sum 303 458
Add 239 394
Sum 137 292
Sub 306 461
Max 307 462
Sub 126 281
Max 245 400
Sub 202 357
Max 68 223
Add 299 454
Max 82 237
Add 171 326
Max 282 437
Sub 248 403
Min 13 168
Sub 186 341
Sum 153 308
Sub 28 183
Max 80 235
Sub 218 373
Sum 1 156
Sub 256 411
Sum 284 439
Sub 240 395
Min 114 269
Sub 266 421
Min 27 182
Sub 110 265
Sum 119 274
Sub 288 443
Sum 266 421
Add 139 294
Max 244 399
Sub 120 275
Min 130 285
Sub 268 423
Max 245 400
Add 61 216
Sum 188 343
Sub 30 185
Sum 77 232
Add 269 424
Max 200 355
Add 45 200
Max 202 357
Sub 244 399
Sum 230 385
Add 41 196
Sum 271 426
Add 27 182
Sum 258 413
Add 303 458
Sum 1 156
Sub 260 415
Max 229 384
Sub 146 301
Sum 243 398
Add 255 410
Min 255 410
Add 47 202
Min 260 415
Sub 238 393
Max 186 341
Add 221 376
Sum 43 198
Add 33 188
Min 206 361
Add 5 160
Max 93 248
Add 79 234
Max 34 189
Sub 202 357
Min 34 189
Add 265 420
Min 246 401
Add 235 390
Sum 194 349
Add 193 348
Min 307 462
Sub 182 337
Sum 220 375
Add 283 438
Sum 171 326
Sub 252 407
Max 228 383
Add 117 272
Sum 83 238
Sub 156 311
Sum 57 212
Sub 24 179
Sum 234 389
Sub 34 189
Max 293 448
Add 137 292
Min 286 441
Sub 22 177
Max 170 325
Add 177 332
Sum 26 181
Sub 294 449
Min 256 411
Sub 60 215
Sum 181 336
Add 289 444
Max 10 165
Sub 112 267
Sum 241 396
Sub 236 391
Max 23 178
Sub 266 421
Max 103 258
Sub 274 429
Sum 234 389
Add 273 428
Min 211 366
Add 179 334
Max 75 230
Add 293 448
Sum 52 207
Add 163 318
Min 34 189
Add 285 440
Sum 228 383
Add 13 168
Min 108 263
Add 199 354
Max 66 221
Add 277 432
Min 62 217
Add 91 246
Max 256 411
Add 35 190
Sum 182 337
Add 155 310
Max 9 164
Sub 224 379
Sum 298 453
Add 281 436
Max 145 300
Sub 6 161
Sum 232 387
Add 247 402
Max 279 434
Add 183 338
Max 238 393
Add 175 330
Min 47 202
Sub 312 467
Sum 186 341
Sub 98 253
Sum 193 348
Add 251 406
Sum 101 256
Sub 106 261
Min 295 450
Sub 208 363
Sum 23 178
Sub 46 201
Max 202 357
Sub 84 239
Min 169 324
Add 175 330
Max 150 305
Sub 68 223
Min 47 202
Sub 190 345
Sum 291 446
Add 31 186
Max 156 311
Add 121 276
Min 36 191
Sub 106 261
Max 86 241
Add 1 156
Max 288 443
Add 9 164
Max 207 362
Add 53 208
Sum 38 193
Sub 110 265
Min 182 337
Sub 230 385
Min 79 234
Sub 282 437
Max 54 209
Add 91 246
Sum 207 362
Sub 76 231
Max 169 324
Sub 126 281
Min 191 346
Add 23 178
Min 19 174
Sub 58 213
