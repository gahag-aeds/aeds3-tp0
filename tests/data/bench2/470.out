7475
97
7696
7785
7999
-3
98
7593
97
101
2
8355
-3
0
103
99
98
-6
96
96
7382
8385
98
98
105
105
97
0
8235
104
7606
6908
-4
0
8075
6408
94
-6
93
7141
8144
96
96
6388
6256
6399
7867
93
6211
-12
-12
103
8637
-12
107
108
6
-12
6968
-10
6505
7710
92
8909
8871
6440
98
-11
105
8151
-11
7499
104
6577
104
104
6175
-12
105
8384
2
6687
-6
106
3
95
7964
106
7087
109
7041
99
97
3
8201
7981
8926
-7
8107
101
-2
107
2
7365
105
3
105
100
99
8608
-5
5
107
7161
105
-11
3
